.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   permissions:
+     security-events: write
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

Comment:
Would be great if the actions were pinned to specific hashes.

Reply:
Done!